### DISCUSSION TITLE
Backup channels and status online/offline

### DISCUSSION THREAD

**Discussion by ruiribeiro2020** [datarhei/restreamer#727] [Ideas]
Hey guys,

It would be great for the restreamer to have the option to backup live channels and also their status, that is:

Main channel -> backup 1 -> backup 2

If main channel offline -> enter 2 -> if offline, enter 3

It was a good resource, we didn't need to constantly change channels.

compliments

**Comment by ruiribeiro2010** [comment]
Hello,

I think it would be a good idea for retreamer to have this implementation.
In case of failure of 1 stream, the backup channel was activated
